CONTRIBUTING.md

```diff
@@ -0,0 +1,58 @@
+Guidelines
+==========
+
+Contributions to `earthpy` are welcome. They are likely to
+be accepted more quickly if they follow these guidelines.
+
+At this stage of development, we are developing a set of
+usable wrapper functions that help make working with earth
+systems data easier.
+
+Edits & Updates
+===============
+
+When submitting a change to the repository, please do the following:
+
+- Please first create an issue that covers the item that you'd like to change, update or enhance. Once a discussion has yielded a vote of support for that addition to the package, you are ready to submit a pull request.
+
+When submitting a pull request:
+
+- All existing tests should pass. Please make sure that the test
+  suite passes, both locally and on
+  [Travis CI](https://travis-ci.org/earthlab/earthpy). Status on
+  Travis will be visible on a pull request. If you want to enable
+  Travis CI on your own fork, please read the pandas guidelines link
+  above or the
+  [getting started docs](http://about.travis-ci.org/docs/user/getting-started/).
+
+- New functionality should include tests. Please write reasonable
+  tests for your code and make sure that they pass on your pull request.
+
+- Classes, methods, functions, etc. should have docstrings. The first
+  line of a docstring should be a standalone summary. Parameters and
+  return values should be documented explicitly.
+
+Improving the documentation and testing for code already in `earthpy`
+is a great way to get started if you'd like to make a contribution.
+
+Style
+-----
+
+- GeoPandas supports python 3 (3.2+) with a single
```

Comment:
Is this a carry over from GeoPandas? I though we were just supporting Python 3?

Reply:
Oh goodness. typos. Yes carryover. I thought i had caught that but i've been working too  many late nights perhaps. i can fix it unless you already have?

Reply:
I have not fixed it - just noticed that it seemed out of place!